data-validation/data-prep.js

```diff
@@ -0,0 +1,22 @@
+const _ = require("lodash");
+
+const validate = (data, schema) => {
+    let dataCopy;
+    if (data.length !== undefined) {
+        dataCopy = [...data]
+    } else {
+        dataCopy = {...data}
+    }
```

Comment:
Hm.... so how come we don't need to do the initialize part (setting values to empty strings, 0s, etc.) anymore?

Reply:
the "coerce types" setting in initializing ajv does this. https://github.com/allen-cell-animated/cell-feature-data/pull/36/files